src/styles/blocks/_watch.scss

```diff
@@ -0,0 +1,87 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  margin-top: 50px;
```

Comment:
It's a BEM block, you can't set margins on it

Reply:
yep, you can't add margin to BEM block, remove it